src/platform/crypt_openssl.c

```diff
@@ -34,13 +34,75 @@
 #include "openssl/rsa.h"
 #include "openssl/ssl.h"
 #include "openssl/x509.h"
+#include "openssl/evp.h"
```

Comment:
codecheck build is complaining:
```
/home/vsts/work/1/s/src/platform/crypt_openssl.c:30:1: error: #includes are not sorted properly [llvm-include-order,-warnings-as-errors]
#include "openssl/kdf.h"
^        ~~~~~~~~~~~~~~~
         "openssl/evp.h"
```
So, could you please put these in alphabetical order?

Reply:
Sure.